client/packages/common/src/hooks/useKeyboard/Keyboard.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { Keyboard } from '@capacitor/keyboard';
+import { Capacitor } from '@capacitor/core';
+import { createRegisteredContext } from 'react-singleton-context';
+
+interface KeyboardControl {
+  isOpen: boolean;
+  isEnabled: boolean;
+}
+
+const defaultKeyboardControl: KeyboardControl = {
+  isOpen: false,
+  isEnabled: false,
+};
+
+const KeyboardContext = createRegisteredContext<KeyboardControl>(
+  'keyboard-context',
+  defaultKeyboardControl
+);
+
+const { Provider } = KeyboardContext;
+
+export const KeyboardProvider = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
```

Comment:
NIT: Noticed this pattern recurring in our codebase. Thoughts on creating an interface for it to easily distinguish it as the props type?

Reply:
We actually do have a PropsWithChildren type that we should be using... this was a copy paste that I didn't even see 😅  Also removed by Andrei's changes though!